src/index.ts

```diff
@@ -97,6 +96,12 @@ export {
   CommentEvent,
   ComponentLifeCycleEvent,
   WhoIsOnlineEvent,
+  Comments,
+  CanvasPin,
+  HTMLPin,
+  MousePointers,
+  WhoIsOnline,
```

Comment:
The entry point to components is no longer `@superviz/lib/components`, it is now `@superviz/sdk`, right?

Reply:
It is both. I don't know if this will work in production, but locally, it is working just fine